src/wasm/wasm-type.cpp

```diff
@@ -949,8 +956,9 @@ std::optional<HeapType> HeapType::getSuperType() const {
       case none:
       case exn:
       case noexn:
-      case string:
```

Comment:
Yikes! This doesn't look like it was correct before, since it should have returned `any`. Do you know how this wasn't caught by tests?

Reply:
The test was just wrong, but it agreed with the code :smile: 

See line 1683 in the gtest modified in this PR.